docs/pages/includes/role-spec.mdx

```diff
@@ -111,6 +111,10 @@ spec:
     # best_effort_drop (try to drop the user at session end, if it doesn't
     # succeed, fallback to disabling it).
     create_db_user_mode: keep
+    # Sets the default shell for auto provisioned SSH users. An absolute path to a shell or a name
+    # reachable through the system PATH are both valid values. Only applies when
+    # `create_host_user_mode` is not set to `off`.
```

Comment:
```suggestion
    # create_host_user_mode is not set to off.
```
Markdown isn't rendered in code snippet comments, so we can remove the backticks.